src/components/Personalization/createFetchDataHandler.js

```diff
@@ -20,13 +20,17 @@ export default ({
   mergeQuery,
   processPropositions,
   createProposition,
-  notificationHandler
+  notificationHandler,
+  consent
 }) => {
   return ({ cacheUpdate, personalizationDetails, event, onResponse }) => {
-    if (personalizationDetails.isRenderDecisions()) {
-      hideContainers(prehidingStyle);
-    } else {
-      showContainers();
+    const { state, wasSet } = consent.current();
```

Comment:
do we reach here if consent isn't given or out?

Reply:
Yes because the awaitConsent happens after this.